rfc6962-bis.xml

```diff
@@ -567,7 +577,7 @@ d0 d1   d2 d3           d0 d1   d2 d3  d4 d5</artwork>
           <spanx style="verb">log_id</spanx> is this log's unique ID, encoded in an opaque vector as described in <xref target="log_id"/>.
         </t>
         <t>
-          <spanx style="verb">timestamp</spanx> is equal to the timestamp from the <spanx style="verb">TimestampedCertificateEntryDataV2</spanx> structure encapsulated in the <spanx style="verb">timestamped_entry</spanx>.
+          <spanx style="verb">timestamp</spanx> MUST be equal to the timestamp from the <spanx style="verb">TimestampedCertificateEntryDataV2</spanx> structure encapsulated in the <spanx style="verb">timestamped_entry</spanx>.
```

Comment:
Reverted - the confusion was because of the digitally-signed field of the struct (which seems to confuse everyone to think this is the actual data + signature rather than just a signature over the described structure).